pontis-ui/src/components/AssetsSection.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import AssetCard from "./Asset/AssetCard";
+import { AssetKeys } from "../hooks/oracle";
+
+const SHOW_DEFAULT = 3;
+
+const AssetsSection = () => {
+  const [numToShow, setNumToShow] = useState<number>(SHOW_DEFAULT);
+
+  const buttonStyle =
+    "bg-slate-300 rounded-2xl px-4 py-2 text-slate-900 shadow-lg hover:shadow-xl cursor-pointer";
+
+  return (
+    <section className="w-screen bg-white px-6 pb-40 sm:px-24 md:px-32">
```

Comment:
why not use div?

Reply:
Removed for now. It's not quite clear to me, though, whether we do or don't have semantic blocks here. ([reference](https://stackoverflow.com/questions/6939864/what-is-the-difference-between-section-and-div))

Reply:
The idea is that we want div's everywhere because that gives us a blank canvas for styling - this is why we don't use headers, paragraphs etc. either. We'd prefer to have consistent elements that we style differently. So I would say whether or not we have semantic blocks is not relevant because we don't adhere to HTML tags original purpose